packages/web-shared/ui-kit/Longform/Longform.styles.js

```diff
@@ -7,19 +7,27 @@ import { TypeStyles } from '../Typography';
 const Longform = styled.div`
   ${TypeStyles.BodyText};
   max-width: 700px;
+  margin-bottom: ${themeGet('space.xl')};
```

Comment:
thanks for throwing these style updates in!

Reply:
Yep! Very obvious compared to this before shot:
<img width="669" alt="Screenshot 2024-01-31 at 4 12 11 PM" src="https://github.com/ApollosProject/apollos-embeds/assets/72768221/e4f7a591-85d5-4606-bf18-40c9e3473c30">